pkg/aws/resources_container.go

```diff
@@ -27,6 +27,7 @@ type ResourcesContainer struct {
 	SecurityGroupsList []*aws2.SecurityGroup   `json:"security_groups"`
 	SubnetsList        []*aws2.Subnet          `json:"subnets"`
 	VpcsList           []*aws2.Vpc             `json:"vpcs"`
+	Provider           string                  `json:"provider"`
```

Comment:
We should have the version field here as well.
How about defining a `Metadata` struct in `common pkg, and embed it in the various `ResourceContainers`?